src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -174,9 +173,18 @@ void FormatTableRows(List<ProcessDetails> rows, StringBuilder tableText)
                 FormatTableRows(printInfo, sb);
                 console.Out.WriteLine(sb.ToString());
             }
-            catch (InvalidOperationException ex)
+            catch (Exception ex)
             {
-                console.Out.WriteLine(ex.ToString());
+                if (ex is InvalidOperationException)
+                {
+                    console.Out.WriteLine(ex.ToString());
+                }
+                else if (ex is System.UnauthorizedAccessException)
+                {
+                    Console.Error.WriteLine("The user that ran this tool does not have access to the NamedPipes pseudo-filesystem. As a result, this tool will not work.");
```

Comment:
There are probably other error conditions that would surface as UnauthorizedAccessException so this error message could be misleading in those cases. For example on Linux GetPublishedProcesses() is trying to enumerate the temp directory. I'd suggest we make this change inside the NetCoreClient library at the point we know that we are enumerating named pipes.